README.rst

```diff
@@ -0,0 +1,224 @@
+.. image:: https://travis-ci.org/jaysonsantos/jinja-assets-compressor.svg?branch=master
+    :target: https://travis-ci.org/jaysonsantos/jinja-assets-compressor
+    :alt: Build Status
+
+jinja-assets-compressor
+=======================
+
+A Jinja2 extension to compile and/or compress your assets.
+
+
+Installing
+----------
+
+::
+
+    pip install jac
+
+For LESS and CSS support, install `less <https://www.npmjs.org/package/less>`_::
+
+    npm install -g less
+
+For COFFEE support, install `coffee-script <https://www.npmjs.com/package/coffee-script>`_::
+
+    npm install -g coffee-script
+
+For Sass and SCSS support, install `sass <https://rubygems.org/gems/sass>`_::
+
+    gem install sass
+
+JavaScript minification is built-in using the Python
+`rJsmin <https://pypi.python.org/pypi/rjsmin>`_ package.
+
+When installing on Mac OS X set this shell variable, because jac dependencies
+contain C code::
+
+    export CFLAGS=-Qunused-arguments
+
+
+Usage
+-----
+
+To use it, you just have to put your css or js inside a compress tag
+
+::
+
+    {% compress 'css' %}
+    <style type="text/sass">
+    sass stuff
+    </style>
+    <link rel="stylesheet" type="text/sass" href="file.sass">
+    {% endcompress %}
+
+    {% compress 'js' %}
+    <script type="text/coffeescript">
+    coffee stuff
+    </script>
+    <script type="text/coffeescript" src="file.coffee"></script>
+    {% endcompress %}
+
+
+Configuring Jinja
+-----------------
+
+You just have to create an environment with jac on it and configure output dir,
+static prefix and say where it can find your sources.
+
+::
```

Comment:
Would it be able to render `.. code-block:: python` ?

Reply:
I think so, let's try it in the next release 👍